upcoming-release-notes/1774.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Bugfix
+authors: [[KaiBelmo](https://github.com/KaiBelmo)]
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
authors: [KaiBelmo]
```

when we publish a release o nthe main website the github author links get automatically added. No need to do this manually.